sdk/tests/client/input_selection/nft_outputs.rs

```diff
@@ -585,7 +591,10 @@ fn missing_ed25519_issuer_transition() {
     )
     .select();
 
-    assert!(selected.is_ok());
+    assert!(matches!(
+        selected,
+        Err(Error::UnfulfillableRequirement(Requirement::Issuer(issuer))) if issuer == Address::try_from_bech32(BECH32_ADDRESS_ED25519_1).unwrap()
+    ));
```

Comment:
Now that I think about it, the test was correct. It actually tests on purpose that a missing issuer has no effect on transitions. There is `missing_ed25519_issuer_created` before that does exactly what you are doing now.

Reply:
It wasn't correct, but the input should then also have the issuer already, is that what you mean?